app/code/Magento/Newsletter/Test/Mftf/Section/NewsletterTemplateSection.xml

```diff
@@ -52,5 +52,13 @@
         <element name="InsertTable" type="button" selector=".mce-i-table" />
         <element name="SpecialCharacter" type="button" selector=".mce-i-charmap" />
         <element name="Preview" type="text" selector="//td[contains(text(),'{{var1}}')]//following-sibling::td/select//option[contains(text(), 'Preview')]" parameterized="true" timeout="60"/>
+        <element name="Queue" type="text" selector="//td[contains(text(),'{{var1}}')]//following-sibling::td/select//option[contains(text(), 'Queue Newsletter')]" parameterized="true" timeout="60"/>
+    </section>
+    <section name="NewsletterTemplateSection">
```

Comment:
There should be only one Section per file.
The file name should reflect the Section Name

Reply:
Also Section should be prefixed with the scope it covers: `Admin` or `Storefront`.